packages/api-client/src/api/getPaymentIntent/index.ts

```diff
@@ -0,0 +1,28 @@
+import axios from 'axios';
+import { ApiContext } from '../../types';
+import getCurrentBearerOrCartToken from '../authentication/getCurrentBearerOrCartToken';
+import getAuthorizationHeaders from '../authentication/getAuthorizationHeaders';
+import { Logger } from '@vue-storefront/core';
+
+export default async function getPaymentIntent({ client, config }: ApiContext, methodId) {
```

Comment:
Could you add a return type to avoid having `Missing return type on function` eslint error? As well as type for `methodId`.

Reply:
Updated.